templates/rhdm73-prod-immutable-kieserver.yaml

```diff
@@ -642,8 +647,14 @@ objects:
             value: "${KIE_SERVER_PWD}"
           - name: KIE_SERVER_CONTAINER_DEPLOYMENT
             value: "${KIE_SERVER_CONTAINER_DEPLOYMENT}"
+          - name: MAVEN_MIRROR_URL
+            value: "${MAVEN_MIRROR_URL}"
+          - name: MAVEN_MIRROR_OF
+            value: "${MAVEN_MIRROR_OF}"
```

Comment:
We do need a MAVEN_MIRROR_URL for immutable, as th emirror can be used in the build - I thought it was already there. But why do we need a MAVEN_MIRROR_OF for immutable?